variables.tf

```diff
@@ -48,25 +48,15 @@ variable "folder_id" {
 }
 
 variable "group_name" {
-  description = "A group to control the project by being assigned group_role - defaults to ${project_name}-editors"
+  description = "A group to control the project by being assigned group_role"
   default     = ""
 }
 
-variable "create_group" {
```

Comment:
The deletion of these variables changes the outward facing APIs. Are these variables no longer available or does one have to use `modules/gsuite_enabled` in order to access these variables?